third_party/move/move-compiler-v2/tests/checking/typing/abort_any_type.exp

```diff
@@ -0,0 +1,21 @@
+// ---- Model Dump
```

Comment:
I've rolled back a lot.  I've kept 2 and filed issues since they illustrate some issues which need to be addressed, one an extra Error so is Urgent.